tests/TestCase.php

```diff
@@ -19,19 +23,19 @@ protected function setUp(): void
 
         $this->withoutMix();
         $this->withExceptionHandling();
-       // $this->withFactories(__DIR__ . '/../database/factories');
+        // $this->withFactories(__DIR__ . '/../database/factories');
 
         $this->artisan('migrate')->run();
     }
 
-    /**
-     * @param \Illuminate\Foundation\Application $app
-     * @return array
-     */
-    protected function getPackageAliases($app)
-    {
-        return [
-            'Form' => FormFacade::class,
-        ];
-    }
+//    /**
```

Comment:
I think you can remove also this code. We're not using the HtmlCollective package anymore.

Reply:
Removed the commented out code